pkg/util/ipvs/kernelcheck_linux.go

```diff
@@ -81,9 +81,15 @@ func (r RequiredIPVSKernelModulesAvailableCheck) Check() (warnings, errors []err
 		}
 		if len(builtInModules) != 0 {
 			warnings = append(warnings, fmt.Errorf(
-				"the IPVS proxier will not be used, because the following required kernel modules are not loaded: %v or no builtin kernel ipvs support: %v\n"+
-					"you can solve this problem with following methods:\n 1. Run 'modprobe -- ' to load missing kernel modules;\n"+
-					"2. Provide the missing builtin kernel ipvs support\n", modules, builtInModules))
+				"\nThe IPVS proxier may not be used because the following required kernel modules are not loaded: %v\n"+
+				"or no builtin kernel ipvs support was found: %v.\n"+
+				"However, these modules may be loaded automatically by kube-proxy for you if they are available on your system.\n\n"+
+				"To verify IPVS support:\n\n"+
+				"   Run \"lsmod | grep 'ip_vs\\|nf_conntrack'\" and verify each of the above modules are listed.\n\n"+
+				"If they are not listed, you can use the following methods to load them:\n\n"+
+				"1. For each missing module run 'modprobe $modulename' (e.g., 'modprobe ip_vs', 'modprobe ip_vs_rr', ...)\n"+
+				"2. If 'modprobe $modulename' returns an error, you will need to install the missing module support for your kernel.\n",
+				modules, builtInModules))
```

Comment:
i think we should use https://github.com/lithammer/dedent for this message. which will combine the strings / trim indentation.

already vendored in k8s; example here:
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/util/config/common_test.go#L231